_studio/shared/umc/codec/h265_dec/src/umc_h265_yuv.cpp

```diff
@@ -99,6 +98,12 @@ void H265DecYUVBufferPadded::deallocate()
 void H265DecYUVBufferPadded::Init(const UMC::VideoDataInfo *info)
 {
     VM_ASSERT(info);
+    if (info == nullptr)
+        throw h265_exception(UMC::UMC_ERR_NULL_PTR);
+    VM_ASSERT(info->GetNumPlanes());
+    if (info->GetNumPlanes() == 0)
+        throw h265_exception(UMC::UMC_ERR_NULL_PTR);
```

Comment:
same

Reply:
this check avoid possible referencing from GetPlaneInfo(0)